pkg/client/unversioned/services.go

```diff
@@ -98,3 +99,16 @@ func (c *services) Watch(label labels.Selector, field fields.Selector, resourceV
 		FieldsSelectorParam(field).
 		Watch()
 }
+
+func (c *services) ProxyGet(name, path string, params map[string]string) ([]byte, error) {
```

Comment:
So, how can I make the result more generic? If I execute Do() (even w/o Raw) the whole response will be put into memory. Do you think Stream() will be better?